pkg/apis/v1beta1/ec2nodeclass.go

```diff
@@ -113,6 +113,16 @@ type EC2NodeClassSpec struct {
 	// +kubebuilder:default={"httpEndpoint":"enabled","httpProtocolIPv6":"disabled","httpPutResponseHopLimit":2,"httpTokens":"required"}
 	// +optional
 	MetadataOptions *MetadataOptions `json:"metadataOptions,omitempty"`
+	// Tenancy of the instance. An instance with a tenancy of dedicated runs on single-tenant hardware.
```

Comment:
Are you able to workaround this limitation by having the VPC tenancy be dedicated? Or do you need a mix of dedicated and non-dedicated instances in the VPC?

Reply:
Typically, dedicated tenancy has less available instance capacity in a region than default tenancy. This is a case where default and tenancy was mixed and used within a VPC.

Reply:
@jonathan-innis Also, EKS cluster cannot create in tenancy: dedicated VPC.
> The subnet(s) provided belong to a VPC with unsupported tenancy. Only subnets from VPCs with default tenancy are supported.